monad-loops.cabal

```diff
@@ -3,7 +3,7 @@ version:                0.4.2.1
 stability:              provisional
 license:                PublicDomain
 
-cabal-version:          >= 1.6
+cabal-version:          >= 1.8
```

Comment:
Needed because:

    monad-loops/ [master] > cabal test
    ./monad-loops.cabal has been changed. Re-configuring with most recently used
    options. If this fails, please run configure manually.
    Resolving dependencies...
    Configuring monad-loops-0.4.2.1...
    cabal: The field 'build-depends: monad-loops' refers to a library which is
    defined within the same package. To use this feature the package must specify
    at least 'cabal-version: >= 1.8'.